### ISSUE TITLE
Unable to install v1.23.8 on MacOSX

### ISSUE THREAD

**Issue by maros7** [golangci/golangci-lint#981]
Thank you for creating the issue!

- [x] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [x] Yes, I've included all information below (version, config, etc).

```bash
++ curl -w '%{http_code}' -sL -o /var/folders/03/_5dhj7qx0x7fx_zfql2fmt9h0d8np2/T/tmp.Eb4R9bwl/golangci-lint-1.23.8-darwin-amd64.tar.gz https://github.com/golangci/golangci-lint/releases/download/v1.23.8/golangci-lint-1.23.8-darwin-amd64.tar.gz
+ code=404
+ '[' 404 '!=' 200 ']'
+ log_debug 'http_download_curl received HTTP status 404'
+ log_priority 7
+ test -z 7
+ '[' 7 -le 6 ']'
+ return 0
+ return 1
```

**Comment by ernado** [comment]
Thank you, seems like CI job failed and no binaries were uploaded for osx.

Sorry, I've re-started job and hope that it will be fixed soon.

**Comment by ernado** [comment]
Should be OK now, please check again.
